src/System.Runtime.Caching/tests/AdditionalCacheTests/AdditionalCacheTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System;
+using System.Runtime.Caching;
+
+namespace System.Runtime.Caching.Tests
+{
+    // These are the tests to fill in some of the coverage in ported Mono caching tests
+    public class AdditionalCacheTests
+    {
+        [Fact]
+        public void DisposedCacheTest()
+        {
+            var mc = new MemoryCache("my disposed cache 1");
+            mc.Add("aa", "bb", new CacheItemPolicy());
+            mc.Dispose();
+
+            Assert.Null(mc["aa"]);
+
+            mc = new MemoryCache("my disposed cache 2");
+            CacheEntryRemovedReason reason = (CacheEntryRemovedReason)1111;
+            var cip = new CacheItemPolicy();
```

Comment:
I presume this it to set it to a value which couldn't be oiginated from the callback? Why not use a nullable?

Reply:
I had this same thought when writing it, but decided to keep it aligned with the other MemoryCache tests style. Maybe a bad idea as it didn't really achieve anything but having another less elegant piece of code :)